lib/types/composite_type.js

```diff
@@ -23,11 +23,9 @@ function wrapField(field, value) {
     return null;
   }
 
-  if (field.type === '*') {
-    if (field.hasOwnProperty('requires'))
-       return (value instanceof field.requires) ? value : new field.requires(value);
-    return value;
-  }
+  if (field.hasOwnProperty('requires'))
+    return (value instanceof field.requires) ? value : new field.requires(value);
```

Comment:
will this ever return true, since in this case requires is a function which returns ErrorCondition and not ErrorCondition itself?